ground-core/pom.xml

```diff
@@ -287,14 +287,20 @@
                 <artifactId>docker-maven-plugin</artifactId>
                 <version>0.4.13</version>
                 <configuration>
-                    <imageName>ground-core</imageName>
-                    <dockerDirectory>docker</dockerDirectory>
+                    <imageName>tarp/ground</imageName>
```

Comment:
yeah you could create a ground repo on docker hub or something so that it makes more sense in the quickstart